app/models/team.rb

```diff
@@ -36,4 +36,9 @@ def matches
   def has_multiple_owners?
     owners.length > 1
   end
+
+  def in_event?
+    events.length > 0
```

Comment:
I think we should use `events.exists?` for readability, see: https://apidock.com/rails/ActiveRecord/Base/exists%3F/class